web/urls.py

```diff
@@ -18,6 +37,7 @@
     path("api/token/", TokenObtainPairView.as_view(), name="token_obtain_pair"),
     path("api/token/refresh/", TokenRefreshView.as_view(), name="token_refresh"),
     path("api/token/verify/", TokenVerifyView.as_view(), name="token_verify"),
+    re_path(r'^swagger/$', schema_view.with_ui("swagger", cache_timeout=0), name="schema-swagger-ui"),
```

Comment:
```suggestion
    re_path(r'^api/docs/$', schema_view.with_ui("swagger", cache_timeout=0), name="schema-swagger-ui"),
```

É bom seguirmos o mesmo padrão de URLs da API.

Dúvida: o que é esse `cache_timeout`?

Reply:
O `cache_timeout` serve para  definir o tempo da expiracao do cache do esquema Swagger. Acontece que se ele nao for usado, sera captada a configuracao de cache do Django Rest Framework. Mas pelo que vi voces nao setaram nenhuma chave de timeout para o DRF, nesse caso se faz desnecessario o parametro. So me confirma se nao tem mesmo essa configuracao de cache dentro da maria-quiteria pra eu tirar o cache-timeout.
A nao ser que voce ache interessante a ideia de ter um cache pra nao ser feita uma requisicao sempre que for acessada a rota de docs.

Reply:
Acho que podemos deixar a configuração padrão e resolver sobre o cache a parte, quando a necessidade surgir.